src/libraries/System.Text.Json/src/System/Text/Json/Document/JsonDocument.cs

```diff
@@ -371,10 +372,10 @@ internal bool TextEquals(int index, ReadOnlySpan<byte> otherUtf8Text, bool isPro
         internal string GetNameOfPropertyValue(int index)
         {
             // The property name is one row before the property value
-            return GetString(index - DbRow.Size, JsonTokenType.PropertyName);
+            return GetString(index - DbRow.Size, JsonTokenType.PropertyName)!;
```

Comment:
I think this makes sense since `GetString` cannot return `null` for valid JSON if the token is a property name.

Reply:
I followed, and agree with, "I think this makes sense ... the null literal, and the caller called GetString."

Didn't follow the rest, but I don't think it changes the conclusion on this line.